spec/requests/api/v3/work_packages/index_resource_spec.rb

```diff
@@ -244,7 +244,7 @@
       end
     end
 
-    context "when providing timestamps", with_ee: %i[baseline_comparison] do
+    shared_examples_for "when providing timestamps" do
```

Comment:
Ss it is required for examples, `with_ee: %i[baseline_comparison]` can also stay on shared_example, then also outer `describe "baseline comparison"` is not necessary